authors/apps/articles/tests.py

```diff
@@ -44,6 +53,14 @@ def setUp(self):
                                      self.article, format="json")
         self.response3 = self.client.post('/api/articles/', self.article_with_tags,
                                     format="json")
+
+    def test_can_create_article(self):
+        response = self.client.post('/api/articles/', self.article,
+                                    format="json")
+        self.response1 = self.client.post('/api/articles/',
```

Comment:
@mendozabree Align the brackets to the agreed convention

Reply:
@samuelbwambale,  I think we may need to cerate a chore to refactor the tests because aside from this chunk of code not being part of this PR, other chunks immediately after it also have the same problem with conventions.

cc: @lym